client/src/legacy/entwine/SubsitesTreeDropdownField.js

```diff
@@ -0,0 +1,33 @@
+/* global jQuery */
+jQuery.entwine('ss', ($) => {
+  /**
+   * Choose a subsite from which to select pages.
+   * Needs to clear tree dropdowns in case selection is changed.
+   */
+  $('select.subsitestreedropdownfield-chooser').entwine({
+    onchange() {
+      // TODO Data binding between two fields
```

Comment:
will this TODO get done?

Reply:
Not in this PR.
Existing code, current head of `2` branch:
https://github.com/silverstripe/silverstripe-subsites/blob/fd10b868ec5dfccd8bd0273c92a30892e2af48cb/javascript/SubsitesTreeDropdownField.js#L9